doc/user_guide/configuration/all-options.rst

```diff
@@ -1192,7 +1240,7 @@ Standard Checkers
 
 --spelling-dict
 """""""""""""""
-*Spelling dictionary name. Available dictionaries: af (aspell), am (aspell), ar (aspell), ast (aspell), az (aspell), be (aspell), be_BY (aspell), be_SU (aspell), bg (aspell), bn (aspell), br (aspell), ca (aspell), cs (aspell), csb (aspell), cy (aspell), da (aspell), de (aspell), de_AT (aspell), de_CH (aspell), de_DE (aspell), el (aspell), en (aspell), en_AU (aspell), en_CA (aspell), en_GB (aspell), en_US (aspell), eo (aspell), es (aspell), es_ES (AppleSpell), et (aspell), fa (aspell), fi (aspell), fo (aspell), fr (aspell), fr_CH (aspell), fr_FR (aspell), fy (aspell), ga (aspell), gd (aspell), gl (aspell), gr (aspell), grc (aspell), gu (aspell), gv (aspell), he (aspell), hi (aspell), hil (aspell), hr (aspell), hsb (aspell), hu (aspell), hu_HU (AppleSpell), hus (aspell), hy (aspell), ia (aspell), id (aspell), it (aspell), it_IT (AppleSpell), kn (aspell), ku (aspell), ky (aspell), la (aspell), lt (aspell), lv (aspell), mg (aspell), mi (aspell), mk (aspell), ml (aspell), mn (aspell), mr (aspell), ms (aspell), mt (aspell), nds (aspell), nl (aspell), nl_NL (AppleSpell), nn (aspell), ny (aspell), or (aspell), pa (aspell), pl (aspell), pt_BR (aspell), pt_PT (aspell), qu (aspell), ro (aspell), ru (aspell), rw (aspell), sc (aspell), sk (aspell), sk_SK (aspell), sl (aspell), sr (aspell), srd (aspell), sv (aspell), sv_SE (AppleSpell), sw (aspell), ta (aspell), te (aspell), tet (aspell), tk (aspell), tl (aspell), tn (aspell), tr (aspell), uk (aspell), uz (aspell), vi (aspell), wa (aspell), yi (aspell), zu (aspell).*
+*Spelling dictionary name. Available dictionaries: en_GB (aspell), en_US (hunspell), en_AU (aspell), en (aspell), en_CA (aspell).*
```

Comment:
This is another problematic default value. I'm starting to doubt whether we should in fact include these generated files...

Reply:
It's not really problematic if the generated file change a little (In fact I could have avoided adding this by doing a partial add). The fact is it permits to see the doc depends on the generator's operating system which it should not, so imo it's a good think that it makes problem apparent. (We can fix it later, should be something like "depends on your operating system" or "default to your current interpreter").